wood-48x48/software/vcarve-post-processors/README.md

```diff
@@ -25,3 +42,22 @@
 * Unit: Inches.
 * Associated Post-Processor: Remove any defaults, add the 2 FCCH post-processors.
 * Double-click the FCCH inches post-processor to set is as default
+
+## Aspire
+
+On the top set of tabs, choose `Toolpaths -> Install PostProcessor...`:
+
+![Install PostProcessor](./images/install_postprocessor_1.png)

```

Comment:
Nit: Does it make sense to add "aspire_" to the start of the filenames, so that if we added equivalent files for Vcarve, they'd be easily distinguished?

Reply:
Done